api/base/serializers.py

```diff
@@ -135,6 +137,90 @@ def to_representation(self, value):
         return {'links': {self.link_type: {'href': url, 'meta': meta}}}
 
 
+class JSONAPIHyperlinkedRelatedField(ser.HyperlinkedRelatedField):
```

Comment:
@abought, would it be easier to review if this change was made in a separate PR?